1.20/src/main/java/io/github/axolotlclient/mixin/PlayerEntityMixin.java

```diff
@@ -84,4 +86,12 @@ public PlayerEntityMixin(EntityType<?> entityType, World world) {
 			comboHud.onEntityDamage(this);
 		}
 	}
+
+	@Override
+	public int getArmor() {
```

Comment:
assuming this is for the hud, i'd recommend modifying the hud

Reply:
yes